### PR TITLE
Base64 transaction id decode

### PR DESCRIPTION
**Pull request by Fernando-Granato**
#### List of Changes
Added method to decode Base64 transactionId received after PGS authorization

#### Motivation and Context
Decoded the transactionId received after PGS authorization to poll the transaction-service

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by ciuffagianluca — APPROVED** [review]
LGTM ✌️

File: src/utils/api/response.ts [hunk 1]
```diff
@@ -29,6 +29,8 @@ const config = getConfigOrThrow();
 const retries: number = 10;
 const delay: number = 3000;
 const timeout: Millisecond = config.CHECKOUT_API_TIMEOUT as Millisecond;
+const decode = (transactionId: string): string => Buffer.from(transactionId, 'base64').toString('binary');
+
```
Comment (pietro-tota) [inline src/utils/api/response.ts:33]: The base64 string sent from transactions-service is calculated starting from the bytes of the UUID (high and low bytes) so here you have to build UUID structure again as shared [here](https://playcode.io/1195321)

**Review by pietro-tota — CHANGES_REQUESTED** [review]
Seems that the UUID conversion does not build UUID correctly, see comment

File: src/utils/api/response.ts (hunk 1 above)
Comment (Fernando-Granato) [reply src/utils/api/response.ts:33]: You are absolutely right

Comment (Fernando-Granato) [reply src/utils/api/response.ts]: @pietro-tota  What do you think now 😉

**Review by pietro-tota — APPROVED** [review]
Well done!🚀

**Pull request merged**